.travis.yml

```diff
@@ -73,6 +73,17 @@ jobs:
     - docker tag precice $DOCKER_USERNAME/precice-ubuntu1804.sudo-develop:latest
     - docker push $DOCKER_USERNAME/precice-ubuntu1804.sudo-develop:latest
 
+  - stage: Building preCICE
+    name: "Ubuntu 18.04.sudo.mpich"
+    if: fork = false
+    script:
+    - docker build -f Dockerfile.Ubuntu1804.sudo.mpich -t precice .
+    after_success:
+    - echo "$DOCKER_PASSWORD" | docker login -u "$DOCKER_USERNAME" --password-stdin
+    - docker images
```

Comment:
You can reduce building and tagging to one command, as fixed here #76

Reply:
7fa22efc1ca465ac6a7c0e590874382ac5b3f2df fixes this